src/screens/RecoveryTransactionList/RecoveryTransactionListScreen.tsx

```diff
@@ -124,12 +133,68 @@ export class RecoveryTransactionListScreen extends PureComponent<Props, State> {
     const isChecked = this.isChecked(transaction);
     const toggle = this.toggleTransaction(isChecked, transaction);
     return (
-      <View style={styles.itemWrapper}>
-        <View style={styles.transactionItemContainer}>
-          <TransactionItem onPress={toggle} item={transaction} />
+      <>
+        <View style={styles.itemWrapper}>
+          <View style={styles.transactionItemContainer}>
+            <TransactionItem onPress={toggle} item={transaction} />
+          </View>
+          <CheckBox onPress={toggle} right checked={isChecked} />
         </View>
-        <CheckBox onPress={toggle} right checked={isChecked} />
-      </View>
+        <View style={styles.itemWrapper}>
+          <View style={styles.transactionItemContainer}>
+            <TransactionItem onPress={toggle} item={transaction} />
+          </View>
+          <CheckBox onPress={toggle} right checked={isChecked} />
+        </View>
+        <View style={styles.itemWrapper}>
+          <View style={styles.transactionItemContainer}>
+            <TransactionItem onPress={toggle} item={transaction} />
+          </View>
+          <CheckBox onPress={toggle} right checked={isChecked} />
+        </View>
+        <View style={styles.itemWrapper}>
+          <View style={styles.transactionItemContainer}>
+            <TransactionItem onPress={toggle} item={transaction} />
+          </View>
+          <CheckBox onPress={toggle} right checked={isChecked} />
+        </View>
+        <View style={styles.itemWrapper}>
+          <View style={styles.transactionItemContainer}>
+            <TransactionItem onPress={toggle} item={transaction} />
+          </View>
+          <CheckBox onPress={toggle} right checked={isChecked} />
+        </View>
+        <View style={styles.itemWrapper}>
+          <View style={styles.transactionItemContainer}>
+            <TransactionItem onPress={toggle} item={transaction} />
+          </View>
+          <CheckBox onPress={toggle} right checked={isChecked} />
+        </View>
```

Comment:
many duplicates

Reply:
good to catch my fault need this for make list longer :)